src/style/index.styl

```diff
@@ -262,3 +256,9 @@ body.aframe-inspector-opened
     a.hover
       background-color #1faaf2
       color #fff
+
+  .toggle-sidebar.left
+      left 0
+
+  .toggle-sidebar.right
+      right 0
```

Comment:
Previous rule matched element with "right" class as a child of element having "toggle-sidebar" class. This is not what we want here. The div has both the "toggle-sidebar" and "right" classes, so we want a ".toggle-sidebar.right" rule without space. If you put a space, it means the same as the previous rule.